docs/ref/contrib/postgres/expressions.txt

```diff
@@ -0,0 +1,29 @@
+PostgreSQL specific query expressions
+=====================================
+
+.. currentmodule:: django.contrib.postgres.expressions
+
+``ArraySubquery``
+=================
+
+.. class:: ArraySubquery(queryset)
+
+``ArraySubquery`` is a ``Subquery`` subclass that uses the PostgreSQL ``ARRAY``
```

Comment:
I would chop `subclass` and add a link to the `Subquery` docs.

Reply:
Indeed!